ush/config.sh.RRFS_CONUS_3km

```diff
@@ -0,0 +1,161 @@
+BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}0000 ${DATE_LAST_CYCL}2300 06:00:00"
+PROD_CYCLEDEF="${DATE_FIRST_CYCL}0300 ${DATE_LAST_CYCL}2300 01:00:00"
+POSTPROC_CYCLEDEF="${DATE_FIRST_CYCL}0300 ${DATE_LAST_CYCL}2300 01:00:00"
+POSTPROC_LONG_CYCLEDEF="${DATE_FIRST_CYCL}0300 ${DATE_LAST_CYCL}2300 03:00:00"
+#ARCHIVE_CYCLEDEF="${DATE_FIRST_CYCL}0700 ${DATE_LAST_CYCL}2300 24:00:00"
+if [[ $DO_SPINUP == "TRUE" ]] ; then
+  SPINUP_CYCLEDEF="00 03-08,15-20 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+fi
+
+FCST_LEN_HRS="3"
+FCST_LEN_HRS_SPINUP="2"
+POSTPROC_LEN_HRS="3"
+POSTPROC_LONG_LEN_HRS="12"
+#FCST_LEN_HRS_CYCLES=(48 18 18 18 18 18 48 18 18 18 18 18 48 18 18 18 18 18 48 18 18 18 18 18)
+for i in {0..23}; do FCST_LEN_HRS_CYCLES[$i]=3; done
+for i in {0..23..3}; do FCST_LEN_HRS_CYCLES[$i]=12; done
+DA_CYCLE_INTERV="1"
+RESTART_INTERVAL="1"
+
+NCORES_RUN_ANAL=240
+HYBENSMEM_NMIN=66
+HALO_BLEND=10
+
+if [[ -n $RESERVATION ]] ; then
+  NNODES_MAKE_ICS="3"
+  PPN_MAKE_ICS="20"
+  NNODES_MAKE_LBCS="3"
+  PPN_MAKE_LBCS="20"
+  NNODES_RUN_POST="1"
+  PPN_RUN_POST="40"
+fi
+
+WTIME_RUN_FCST="02:45:00"
+WTIME_MAKE_LBCS="02:00:00"
+
+EXTRN_MDL_NAME_ICS="FV3GFS"
+EXTRN_MDL_NAME_LBCS="FV3GFS"
+
+FV3GFS_FILE_FMT_ICS="grib2"
+FV3GFS_FILE_FMT_LBCS="grib2"
+
+envir="para"
+
+NET="RRFS_CONUS"
+TAG="RRFS_CONUS_3km"
+
+USE_CUSTOM_POST_CONFIG_FILE="TRUE"
+TESTBED_FIELDS_FN="testbed_fields_bgdawp.txt"
+CUSTOM_POST_CONFIG_FP="$(cd "$( dirname "${BASH_SOURCE[0]}" )/.." &>/dev/null&&pwd)/fix/upp/postxconfig-NT-fv3lam_rrfs.txt"
+CUSTOM_POST_PARAMS_FP="$(cd "$( dirname "${BASH_SOURCE[0]}" )/.." &>/dev/null&&pwd)/fix/upp/params_grib2_tbl_new"
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_dev1"
+NCARG_ROOT="/apps/ncl/6.5.0-CentOS6.10_64bit_nodap_gnu447"
+NCL_HOME="/home/rtrr/RRFS/graphics"
+NCL_REGION="conus"
+MODEL="RRFS_conus_3km"
+
+#
+# In NCO mode, the following don't need to be explicitly set to "FALSE" 
+# in this configuration file because the experiment generation script
+# will do this (along with printing out an informational message).
+#
+#RUN_TASK_MAKE_GRID="FALSE"
+#RUN_TASK_MAKE_OROG="FALSE"
+#RUN_TASK_MAKE_SFC_CLIMO="FALSE"
+
+RUN="RRFS_conus_3km"
+COMINgfs=""
+
+. set_rrfs_config.sh
+
+STMP="YourOwnSpace/stmp"  # Path to directory STMP that mostly contains input files.
+PTMP="YourOwnSpace/ptmp"  # Path to directory STMP that mostly contains input files.
+NWGES="YourOwnSpace/nwges"  # Path to directory NWGES that save boundary, cold initial, restart files
+
+#
+# In NCO mode, the user must manually (e.g. after doing the build step)
+# create the symlink "${FIXrrfs}/fix_sar" that points to EMC's FIXLAM
+# directory on the machine.  For example, on hera, the symlink's target
+# needs to be
+#
+#   /scratch2/NCEPDEV/fv3-cam/emc.campara/fix_fv3cam/fix_sar
+#
+# The experiment generation script will then set FIXLAM to 
+#
+#   FIXLAM="${FIXrrfs}/fix_lam/${EMC_GRID_NAME}"
+#
+# where EMC_GRID_NAME has the value set above.
+#
+
```

Comment:
Do we want to remove line 147-161?  It is not relevant now and sometime causes confusions.

Reply:
removed